### PR TITLE
remove SafeMetricsReporter from README.md

### PR DESCRIPTION
**Pull request by malafeev**
fix for #16

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,14 +26,21 @@ the reported metrics will be ignored at the backend.
...
+  Sender<?,?> sender = new OkHttpSender(
       "MyServiceName", // Service/Component name.
-      "https://" // Metrics url
       "MyAccessToken", // Access Token. Nullable
       "1.2.3", // Service version. Nullable
+      "https://" // Metrics url
+      30000 // connect timeout in milliseconds
+  );
+
+  Metrics metrics = new Metrics(
```
Comment (carlosalberto) [inline README.md:42]: Can we mention `Metrics` inherits from `Thread`? This way it makes more sense as to why users should keep a handle to it.

Comment (malafeev) [reply README.md:42]: // Metrics inherits from Thread.
2 line below

Comment (carlosalberto) [reply README.md:42]: Can we move that line here? ;)

Comment (malafeev) [reply README.md:42]: done

**Pull request merged**